lighthouse-core/report/html/renderer/psi.js

```diff
@@ -126,6 +139,13 @@ function _getFinalScreenshot(perfCategory) {
   return details.data;
 }
 
+// Defined by lib/file-namer.js, but that file does not exist in PSI. PSI doesn't use it, but
+// needs some basic definition so report-ui-features.js typechecks in PSI.
+// @ts-expect-error
+// eslint-disable-next-line no-unused-vars
+function getFilenamePrefix(lhr) {
+}
```

Comment:
was this from an old version of the change? There's no change in the dependency tree now so this shouldn't be necessary

Reply:
report-ui-features calls `getFilenamePrefix`, which fails the internal BUILD.

Reply:
i _think_ its cuz reportuifeatures wasn't being used in PSI renderer before, but now is. 
and closure is whining about the use of this fn from reportuifeatures

Reply:
> but now is.

ah, yeahh, that's an important bit :)

next renderer roll will add it

Reply:
Ah got it, I just assumed tsc. Maybe add "closure compiler" to the comment somewhere? Not what we usually mean by typechecking and I'll probably be confused again the next time I come across this :)